chat-plugins/github.js

```diff
@@ -0,0 +1,96 @@
+/*
+	Github plugin by Spandamn
+	This is based on https://github.com/Ecuacion/Pokemon-Showdown-Node-Bot/blob/master/features/github/index.js
+	Yes, some of the code is copied so credits to xfix, Ecuacion, jd, and anyone who's name I forgot to put here
+*/
+'use strict';
+
+let config = {};
+
+if (!Config.github) {
+	return;
+} else if (Config.github === {}) {
+	config.port = 3420;
+	config.secret = "";
+} else {
+	config = {
+		port: Config.github.port,
+		secret: Config.github.secret,
+	};
+}
+
+const git = exports.github = require('githubhook')(config);
+
+let updates = {};
+
+let sendReport = function (html) {
+	if (Config.github && Config.github.rooms) {
+		for (let curRoom of Config.github.rooms) {
+			let room = Rooms(curRoom);
+			if (!room) return;
+			room.add(`|html|<div class="infobox">${html}</div>`);
+			room.update();
```

Comment:
These three lines can be turned into one:
``if (room) room.add(`|html|<div class="infobox">${html}</div>`).update();``